ci/jenkins/Jenkins_steps.groovy

```diff
@@ -992,6 +1006,25 @@ def test_centos7_python3_cpu() {
     }]
 }
 
+def test_centos7_python3_cpu_mkldnn() {
```

Comment:
Why we need new test for MKL-DNN default? I thought `USE_MKLDNN=1` is well tested in the existing CI.

Reply:
@marcoabreu Any thoughts?

Reply:
@TaoLv MKLDNN is well-tested on CI for Unix Python, but not for CentOS Python.

Reply:
Agree with Tao. I also don't think that CentOS should make any difference for mkldnn, so there's no need to test that path. Wdyt, @TaoLv

Reply:
@mseth10 Do you know any problem with CentOS Python which is not exposed in the existing CI? If has, I would like to have another PR for adding it into CI and keep this PR focusing on MKL-DNN build change.

Reply:
@TaoLv CentOS Python tests on CI use non MKL-DNN builds. Shouldn't we be using MKL-DNN builds now that MKL-DNN is default on CentOS? There is no other problem exposed.

Reply:
@TaoLv Just to note, I checked and the CI execution time is not increased, as the tests run in parallel and Ubuntu test takes longer.

Reply:
I don't think it is necessary to be honest. Mkldnn should not really interface with the operating system, but I'm not certain. Ideally Tao could consult with some Intel engineers. But to unblock the progress of this PR, we should have that discussion in a separate PR.

Reply:
Thanks for the feedback. For speedy progress of this PR, I will not add tests to CI as part of it.